swingset/src/main/java/com/nqadmin/swingset/utils/SSComponentInterface.java

```diff
@@ -145,7 +146,9 @@ default void configureTraversalKeys() {
 		final Set<AWTKeyStroke> forwardKeys = ((JComponent) this)
 				.getFocusTraversalKeys(KeyboardFocusManager.FORWARD_TRAVERSAL_KEYS);
 		final Set<AWTKeyStroke> newForwardKeys = new HashSet<>(forwardKeys);
-		newForwardKeys.add(KeyStroke.getKeyStroke(KeyEvent.VK_ENTER, 0));
+		if (!(this instanceof JTextArea)) {
+			newForwardKeys.add(KeyStroke.getKeyStroke(KeyEvent.VK_ENTER, 0));
+		}
```

Comment:
It's been a very long time since I've looked at keyboard focus traversal. So enter was causing problems only with the JTextComponent?

I thought changing keyRelease to keyPressed took care of the issue. Ah, so this change means that enter when in a textcomponent, goes nowhere?

Reply:
Not any text component, just JTextArea, which are the multiline text boxes. Otherwise you can use Enter to add line breaks.